near2eth/near2eth-block-relay/index.js

```diff
@@ -267,7 +267,7 @@ class Near2EthRelay {
         }
         delay = Math.max(delay, minDelay)
         console.log(
-          `Client height is ${bridgeState.currentHeight}, chain height is ${lastBlock.inner_lite.height}. Sleeping for ${delay} seconds.`
+          `Last valid header on the client: ${bridgeState.currentHeight}. Waiting for block ${lastBlock.inner_lite.height} to become valid for ${delay} seconds.`
```

Comment:
This message is not accurate either. `lastBlock.inner_lite.height` isn't necessarily the same as the height of the current unverified block. Rather, it's the height of the block that could have been submitted if there were no unverified block. These values are usually the same during catchup (both correspond to the last block in the epoch that follows the epoch of the current verified block), but not during normal operation.